Mathlib/Data/Finset/Max.lean

```diff
@@ -539,3 +539,14 @@ theorem isLUB_mem [LinearOrder α] {i : α} (s : Finset α) (his : IsLUB (s : Se
   @isGLB_mem αᵒᵈ _ i s his hs
 
 end Finset
+
+theorem Multiset.exists_max_image {α R : Type*} [DecidableEq α] [DecidableEq R] [LinearOrder R]
```

Comment:
Can you check whether you need `DecidableEq` for the *statement*? If not, please remove and use `classical` in the proof.